test_package/conanfile.py

```diff
@@ -1,26 +1,26 @@
+from pathlib import Path
 from conans import ConanFile, CMake
 
 
 class DummyTestConan(ConanFile):
     settings = "os"
     generators = "cmake_find_package"
-    requires = "dummy/1.0@demo/testing"
-
-    def configure_cmake(self):
-        cmake = CMake(self)
-        cmake.configure()
-
-        return cmake
 
     def build(self):
         self.cmake = CMake(self)
 
-        print("{}".format(self.cmake.command_line))
+        prefix = self.cmake.definitions.get("CMAKE_PREFIX_PATH", "")
```

Comment:
I would rely on Python functionality as Conan already conforms to certain Python features.
In this case dictionary (dict).

In your ```test_package/conanfile.py``` (line: 12):
```python
prefix = []
if ("CMAKE_PREFIX_PATH" in self.cmake.definitions): prefix = self.cmake.definitions["CMAKE_PREFIX_PATH"]
self.output.info("CMAKE_PREFIX_PATH == " + prefix)
deps_build_paths = [dep_build_path for dep in self.deps_cpp_info.deps for dep_build_path in self.deps_cpp_info[dep].build_paths]
self.cmake.definitions["CMAKE_PREFIX_PATH"] = ";".join([str(Path(x).as_posix()) for x in (prefix.split(";") + deps_build_paths)])
```

Reply:
Right, thanks for the tips!